core/bignum/abs_spec.rb

```diff
@@ -1,7 +0,0 @@
-require File.expand_path('../../../spec_helper', __FILE__)
```

Comment:
I think your commit cc6dc2b2db8aa4abdf1eaaaca95d707bbd0129f0 is inadvertently part of this PR.
Could you remove it?
(One way is `git rebase -i HEAD~3`, or new branch + cherry pick the other commits + reset this branch to the new branch and finally `git push -f`.)